src/immutably-array/move.js

```diff
@@ -0,0 +1,14 @@
+import immutably from 'immutably';
+
+function move(input, path, fromIndex, toIndex) {
+    return immutably.apply(input, path, (input) => {
+        const output = immutably.clone(input);
```

Comment:
I think there should be a condition for when `fromIndex` and `toIndex` are the same.